public/app/features/datasources/components/DataSourceTestingStatus.tsx

```diff
@@ -169,6 +181,17 @@ export function DataSourceTestingStatus({ testingStatus, exploreUrl, dataSource
               ) : null}
             </>
           )}
+          {severity === 'error' && links.length > 0 && (
+            <div className={styles.linksContainer}>
+              {links.map((link) => {
+                return (
+                  <a key={link.id} href={link.path} onClick={link.onClick} className={styles.pluginLink}>
```

Comment:
Should we throw in a 
```suggestion
                  <a key={link.id} href={sanitizeUrl(link.path)} onClick={link.onClick} className={styles.pluginLink}>
```
here? Not sure if there would be any vectors, might be not needed.